app/views/users/registrations/new.html.erb

```diff
@@ -24,7 +24,7 @@
 
       <div>
         <h4 class="heading-4" >Date of birth</h4>
-        <p class="text-sm text-dark-300">This will not be shown publicly. Confirm your own age, even if this account is for as business, a pet, or something else.</p>
+        <p class="description-2">This will not be shown publicly. Confirm your own age, even if this account is for as business, a pet, or something else.</p>
```

Comment:
Extra s in `for as business`
```suggestion
        <p class="description-2">This will not be shown publicly. Confirm your own age, even if this account is for a business, a pet, or something else.</p>
```